vendor/k8s.io/kubernetes/staging/src/k8s.io/apimachinery/pkg/api/meta/patch.go

```diff
@@ -0,0 +1,13 @@
+package meta
+
+import "k8s.io/apimachinery/pkg/runtime/schema"
+
+var oapiResources = map[schema.GroupVersionResource]bool{
+	{Resource: "deploymentconfig"}:  true,
+	{Resource: "deploymentconfigs"}: true,
+	{Resource: "dc"}:                true,
```

Comment:
there's less than 100 of these

Reply:
can we make this pluggable in upstream, so you can register "ignored" resources like:

```golang
type predicateFunc func(gvk) bool
var IgnoredResources = map[schema.GroupVersionResource]predicateFunc{}

// then in origin we will have something like:

meta.IgnoredResources[{Resource: "deploymentconfig"}] = func(gvk) { len(gvk.Group) == 0 }

```

Reply:
(just thinking loud to avoid carry patch ;-)

Reply:
> can we make this pluggable in upstream, so you can register "ignored" resources like

I don't think there's a use-case upstream.